lib/utils/addr.go

```diff
@@ -266,26 +267,10 @@ func IsLocalhost(host string) bool {
 	return ip.IsLoopback() || ip.IsUnspecified()
 }
 
-// IsLoopback returns 'true' if a given hostname resolves to local
+// IsLoopback returns 'true' if a given hostname *only* resolves to local
 // host's loopback interface
 func IsLoopback(host string) bool {
```

Comment:
Let's fully replace this one with apiutils.IsLoopback, keeping this only adds to the codebase.

Reply:
Done